source/code-generation/query-converter-test-bench.txt

```diff
@@ -10,10 +10,43 @@ Query Converter Test Bench
    :depth: 1
    :class: singlecol
 
-Stub page.
+Use Query Converter Test Bench to test SQL queries 
+and stored procedures converted by :ref:`rm-query-converter`. 
+Query converter test bench uses the connections defined for your 
```

Comment:
[Capitalization]
This applies throughout since it is the product name.

```suggestion
Query Converter Test Bench uses the connections defined for your 
```